### PR TITLE
[Plagiarism Detector] Plagiarism table

### PR DESCRIPTION
**Pull request by MohitOhlyan**
# Descripción

Added a new table to store the plagiarism scores obtained from running copydetect. 

Fixes: #6688 

# Comentarios
Sorry for closing the Previous PR on this issue. I was having some weird git issues in my local branch that was making me not able to push to the remote branch. 

Please let me know if there can be any more modification and if there are any errors. 

# Checklist:

- [x] El código sigue la [guía de
      estilo](https://github.com/omegaup/omegaup/wiki/Coding-guidelines) de
      omegaUp.
- [x] Se corrieron todas las pruebas y pasaron.
- [ ] Si se está agregando funcionalidad nueva, se agregaron pruebas.
- [x] Si el cambio es grande (> 200 líneas), hay que intentar partirlo en
      varios pull requests. De preferencia uno para los controladores + phpunit
      y luego otro para la interfaz.

### CODE REVIEW

File: frontend/database/00208_plagiarism_table.sql [hunk 1]
```diff
@@ -0,0 +1,13 @@
+CREATE TABLE IF NOT EXISTS `Plagiarisms` (
+    `plagiarism_id` int(11) NOT NULL AUTO_INCREMENT COMMENT 'El identificador único para cada potencial caso de plagio',
+    `contest_id` int(11) NOT NULL,
+    `submission_id_1` int(11) NOT NULL COMMENT 'El identificador del envío del primer código plagiado',
+    `submission_id_2` int(11) NOT NULL COMMENT 'El identificador del envío del segundo código plagiado',
+    `score_a` tinyint(1) NOT NULL COMMENT 'porcentaje de plagio encontrado usando copydetect en el envío 1',
+    `score_b` tinyint(1) NOT NULL COMMENT 'porcentaje de plagio encontrado usando copydetect en el envío 2',
```
Comment (lhchavez) [inline frontend/database/00208_plagiarism_table.sql:7]:
```suggestion
    `score_1` tinyint(1) NOT NULL COMMENT 'porcentaje de plagio encontrado usando copydetect en el envío 1',
    `score_2` tinyint(1) NOT NULL COMMENT 'porcentaje de plagio encontrado usando copydetect en el envío 2',
```

for consistency with the submissions.

File: frontend/database/00208_plagiarism_table.sql [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    `contents` TEXT NOT NULL COMMENT 'Almacena los puntajes de similitud de los envíos y el rango de números de línea de esos puntajes',
```
Comment (lhchavez) [inline frontend/database/00208_plagiarism_table.sql:8]:
```suggestion
    `contents` TEXT NOT NULL COMMENT 'Almacena los rangos de números de línea de las similitudes',
```

**Review by lhchavez — COMMENTED** [review]
just small details and we're good!

File: frontend/database/00208_plagiarism_table.sql (hunk 1 above)
Comment (MohitOhlyan) [reply frontend/database/00208_plagiarism_table.sql:7]: Got it.

File: frontend/server/src/DAO/Base/Plagiarisms.php [hunk 3]
```diff
@@ -0,0 +1,320 @@
+<?php
```
Comment (lhchavez) [inline frontend/server/src/DAO/Base/Plagiarisms.php:1]: can the php files be deleted? database schema changes should land independently of code changes.

Comment (MohitOhlyan) [reply frontend/server/src/DAO/Base/Plagiarisms.php:1]: Whenever I run the dao_schema script these changes are included and when pushing I get asked to run another command which fixes these php changes and VO and DAO changes but this particular Plagiarism.php didn't change. 

Should I manully delete this file?

Comment (lhchavez) [reply frontend/server/src/DAO/Base/Plagiarisms.php:1]: yes please, don't run that script yet because `dao_schema.sql` is not yet to be modified.

**Review by lhchavez — APPROVED** [review]

**Pull request merged**